dev/AppLifecycle/AppLifecycle.idl

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.ProjectReunion
```

Comment:
@jonwis , I was imagining that we would not have anything in "Microsoft.ProjectReuion" namespace.  Is this OK?

Reply:
For now it is, but we should probably move this to Ms.AppModel.Something eventually